Documentation/ApiOverview/Seo/_ExampleWebsiteTitle/_WebsiteTitleProvider.php

```diff
@@ -13,26 +13,33 @@
 #[Autoconfigure(public: true)]
 final readonly class WebsiteTitleProvider implements PageTitleProviderInterface
 {
+    protected ServerRequestInterface $request;
```

Comment:
```suggestion
    private ServerRequestInterface $request;
    
```

As the class is final and protected does not make sense. Additionally, add an empty line after for better readability.